src/testdrive/src/action.rs

```diff
@@ -494,6 +496,9 @@ pub(crate) async fn build(
                     "schema-registry-wait-schema" => Box::new(
                         schema_registry::build_wait(builtin, context.clone()).map_err(wrap_err)?,
                     ),
+                    "skip-if" => Box::new(
```

Comment:
Do you mind adding a short mention of this to the `doc/develper/testdrive.td` file? Thanks!